snapshot/src/main/java/io/camunda/zeebe/snapshots/ConstructableSnapshotStore.java

```diff
@@ -19,9 +19,9 @@
    * @param term the term to which the snapshots corresponds to
    * @param processedPosition the processed position in the snapshot
    * @param exportedPosition the exported position in the snapshot
-   * @return an optional with a transient snapshot if new transient snapshot was taken successfully,
-   *     otherwise return an empty optional
+   * @return an Either that contains and exception or transientSnapshot if it was taken
```

Comment:
:thought_balloon: could be shorten like this
```suggestion
   * @return either an exception or transientSnapshot, if it was taken
```

Reply:
Thanks. I was struggling to write this :smile: